0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -148,46 +168,49 @@ Content-Type: application/json
-* The `amount` is determined by the sender:
+The sender uses the receiver details to create the PSK2 payment:
 
-    * The sender uses the `ledger_info.currency_scale` to determine the integer amount to include in the ILP packet (e.g. if they want the receiver to receive 10 units of `ledger_info.currency_code` on their ledger, a `ledger_info.currency_scale` value of 2 would indicate they should set the `amount` to 1000). In cases where the sender knows the integer amount the receiver should get, for example if that amount was requested by the receiver out of band, the sender does not need to use `ledger_info.currency_code` and `ledger_info.currency_scale`
-    * The sender SHOULD NOT set an amount greater than the `maximum_destination_amount` or lower than the `minimum_destination_amount`, as this will be rejected by the receiver
+* The `destination_account` should be used as the PSK2 destinationAccount.
+* The `shared_secret` should be used as the PSK2 sharedSecret.
+* If present, the `balance.maximum` SHOULD be used as the PSK2 chunked payment amount.
 
-* The `data` is encoded using the [Pre-Shared Key protocol](../0016-pre-shared-key/0016-pre-shared-key.md):
+In a UI, the `ledger_info` and `receiver_info` objects (if present) can be used for display purposes. These objects can be manipulated by the receiver in any way, so amounts SHOULD be displayed in source units when possible.
 
-    * The key is the `shared_secret` provided by the receiver
-    * Additional data may be included in the PSK details. SPSP data MUST be JSON.
-    * Private PSK headers SHOULD include `Content-Type: application/json` and `Content-Length: <byte length of data>`
+Note that the sender can send as many PSK2 payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
 
-Note that the sender can send as many payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
+## Appendix A: Payment Pointer
 
-## Appendix A: (Optional) Webfinger Discovery
+This is the recommended way to identify an SPSP receiver, and is intended to be the main form of identifier that users on Interledger will interact with. It can be used as a persistent identifier for a person or as a temporary identifier to represent an invoice, much like a bitcoin address.
 
-Whenever possible, receiver URLs should be exchanged out-of-band and discovery should be skipped. However, in some cases, it may be useful to have a standardized user-friendly identifier. This discovery method describes how to resolve such an identifier to an SPSP endpoint.
+The payment pointer is in the form `$example.com/bob` (A payment pointer with no path is also acceptable, i.e. `$bob.example.com`). This is converted into an SPSP URI.
```

Comment:
I kind of like the `$example...` format, but having it start with a dollar sign makes it quite US-centric. It's unfortunate that there is no widely-recognized symbol that just means "money" or "currency" without being tied to a specific one.

Reply:
Most keyboards have the `$`, at least. It also has the connotation of being a variable that expands to something else, which is cool (it 'expands' to your ILP address)

Reply:
How do you feel about the name "Payment Pointer" for it?

Reply:
Typical American. You adopt the `$` sign for your currency and suddenly it's "US-centric". ;)

The most common hypothesis for the origin of `$` according to Wikipedia:

> The symbol was adopted by Charles V and was part of his coat of arms representing Spain's American possessions. The symbol was later stamped on coins minted in gold and silver. The coin, also known as Spanish dollar, was the first global currency used in the entire world since the Spanish Empire was the first global Empire.

So, if anything it's Spanish Empire-centric, not US-centric.

As a result, no other symbol is used by as many different currencies:

> Australian (A$), Bahamian (B$), Barbadian (Bds$), Belizean (BZ$), Bermudian (BD$), Brunei (B$), Canadian (Can$), Cayman Islands (CI$), East Caribbean (EC$), Fiji (FJ$), Guyanese (G$),[5] Hong Kong (HK$/元/圓), Jamaican (J$), Kiribati, Liberian (L$), Linden Dollar (Second Life virtual world) (L$ or LD$), Namibian (N$), New Zealand (NZ$), Singaporean (S$), Solomon Islands (SI$), Surinamese (SRD), Taiwanese (NT$/元/圓), Trinidad and Tobago (TT$), Tuvaluan, United States (US$), and Zimbabwean (Z$) dollars
>
> Argentine, Chilean (CLP$), Colombian (COL$), Cuban ($MN), Cuban convertible (CUC$), Dominican (RD$), Mexican (Mex$), and Uruguayan ($U) pesos

It also appears in the currency symbols for Brazilian Real (R$) and Nicaraguan córdoba (C$) among a few others. That means there are people on every continent who recognize `$` as the symbol for *their* currency. No other currency symbol comes close. You could make a case for `¥` which I would guess is the currency symbol used by the most people. But it is used only by two currencies: Chinese Renminbi yuan and Japanese yen, so I don't think you can make a strong case that it is the best global choice. There is the "generic currency sign" `¤`, but *nobody* would recognize that thing.

There are of course also pragmatic reasons for using `$`. It's an IA5 character and therefore easily encoded and included in pretty much all [keyboard layouts](https://en.wikipedia.org/wiki/Keyboard_layout) worldwide. We should check if that is still true for mobile input methods.

Is the `$` perfect? No. It's not. Ideally, something like `¤` would be an IA5 character and widely recognized. But we're in the real world and out of the available choices, the `$` has a huge lead. Someone patriotic whose country uses a different symbol might think it's associated with the US and feel excluded. If anyone has a solution for that, let me know. But I also think there will be a lot of people that wouldn't react that way. Speaking as a German, the `$` to me just means money. The fact that the US adopted it is just another example of a young country that's perhaps lacking a little bit of cultural identity. And the fact that Americans would be worried that the `$` is not culturally neutral is an example of maybe a little bit of lack of cultural awareness - the majority of people whose local currency symbol is or includes the `$` are not Americans. The US is just one of several countries that used `$` because they didn't have their own historic currency symbol and `$` was basically the international symbol for money at the time.